src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/DirectoryBasedStoreProvider.cs

```diff
@@ -201,6 +201,8 @@ private void AddCertToStore(ICertificatePal certPal)
 
         public void Remove(ICertificatePal certPal)
         {
+            if (!Directory.Exists(_storePath)) { return; }
```

Comment:
Style: We don't allow block bodies on the same line.


```suggestion
            if (!Directory.Exists(_storePath))
            {
                return;
            }
```

or, since this is a precondition check:

```suggestion
            if (!Directory.Exists(_storePath))
                return;
```